src/NServiceBus.Core/Unicast/Queuing/Installers/ForwardReceivedMessagesToQueueCreator.cs

```diff
@@ -3,7 +3,7 @@ namespace NServiceBus.Unicast.Queuing.Installers
 
     class ForwardReceivedMessagesToQueueCreator : IWantQueueCreated
     {
-        public Address Address{get; private set;}
+        public Address Address{get; set; }
```

Comment:
Sorry, didn't notice there are two separate commits in this PR and thought it has to do something with the message serializer.